test/bin/find.js

```diff
@@ -80,32 +89,47 @@ describe('bin', function() {
     process.argv[2] = '-u'
     proxyquire('../../src/bin/find', stub)
     assert.ok(getUnusedRules.called)
+    process.argv[2] = '--unused'
+    proxyquire('../../src/bin/find', stub)
+    assert.ok(getUnusedRules.called)
   })
 
-  it('options -u|--unused and no unused rules found', function() {
+  it('options -u|-unused and no unused rules found', function() {
     getUnusedRules.returns([])
     process.exit = function(status) {
       assert.equal(status, 0)
     }
     process.argv[2] = '-u'
     proxyquire('../../src/bin/find', stub)
     assert.ok(getUnusedRules.called)
+    process.argv[2] = '--unused'
+    proxyquire('../../src/bin/find', stub)
+    assert.ok(getUnusedRules.called)
   })
 
-  it('option -u|--unused along with -n|no-error', function() {
+  it('option -u|-unused along with -n|-no-error', function() {
     process.exit = function(status) {
       assert.equal(status, 0)
     }
     process.argv[2] = '-u'
     process.argv[3] = '-n'
     proxyquire('../../src/bin/find', stub)
     assert.ok(getUnusedRules.called)
+
+    process.argv[2] = '--unused'
+    process.argv[3] = '-no-error'
```

Comment:
I guess so. 

BTW, do we needs `yargs` or can we leave w/o it?

---

I agree the proposed convention is confusing, but its better than the current one, which is wrong.